tox.ini

```diff
@@ -56,7 +72,7 @@ basepython = python2.7
 deps = {[testenv]basedeps}
        nosegae
 commands =
-    nosetests --with-gae --gae-application=tests/data --logging-level=INFO {posargs} tests/test_appengine.py
+    nosetests --with-gae --gae-application=tests/data --logging-level=INFO tests/test_appengine.py
```

Comment:
OK this SGTM. Would be nice if GitHub let us mark these comments as "resolved". There should be a way to tag a GitHubber in situations like this with feature requests.